docs/user-guide/transformer_expressions.md

```diff
@@ -535,3 +549,397 @@ variables:
 
 Output: `"2021-11-30 15:00:00 +0900 WIT"`
 ```
+
+
+## Column Expression
+Column expression is function that can be invoked by column values in a table
+
+### Get
+`Get` will retrieve a row in column based on the given index
+
+#### Input
+| Name | Description |
+|------|-------------|
+| Index| Position of rows starts with 0 |
+
+#### Output
+Single column row
+
+#### Examples
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Users try to retrieve index 2 for column `restaurant_id`
+
+Standard Transformer Config:
+```
+variables:
+- name: restaurant_2
+  expression: yourTableName.Col("restaurant_id").Get(2)
```

Comment:
Can we use `avg_order_1_day` as example? And also, shouldn't it return `3` in this case since it's 0-index

Reply:
oh yeah 🤦 , I start count from 1